modules/configuration/src/testFixtures/java/org/apache/ignite/internal/configuration/testframework/ConfigurationExtension.java

```diff
@@ -273,7 +274,7 @@ private static Object cfgValue(
         );
 
         // RootKey must be mocked, there's no way to instantiate it using a public constructor.
-        RootKey rootKey = mock(RootKey.class);
+        RootKey rootKey = mock(RootKey.class, withSettings().lenient());
```

Comment:
Without `lenient`, a test (unrelated to this PR) started failing. It happened because I had to add `@ExtendWith(ConfigurationExtension)`.

The mock `RootKey` has to be lenient so that it does not fail a test in which any of the mocked method is not called.

Reply:
I do not quite understand how tests can fall if methods are not called?

Reply:
I added an `@ExtendWith(ConfigurationExtension.class)` for `org.apache.ignite.internal.tx.storage.state.test.TestTxStateStorageTest` and nothing fell in my pr.

Reply:
Take a look at `ItRaftGroupServiceTest`: it will fail if you remove `lenient()`. https://ci.ignite.apache.org/buildConfiguration/ApacheIgnite3xGradle_Test_RunAllTests/6982075?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&pluginCoverage=true&expandCode+Inspection=true&expandBuildChangesSection=true&expandBuildTestsSection=true

Reply:
It turns out interesting, locally I can not reproduce.

Reply:
It reproduces easily on my machine in this branch